src/main.c

```diff
@@ -222,7 +230,16 @@ int main(int argc, char *argv[])
 
     if (invert_reset)
     {
-        mode |= MODE_INVERT_RESET;
+        if (mode == 4)
+        {
+            mode = 4;
```

Comment:
This code is only ever run if `mode == 4` so assigning it to 4 is unnecessary.

Reply:
Well, crap.  See: https://github.com/msalau/rl78flash/blob/29969af6f7a496ccb928b02e648e6e48157acc68/src/rl78.h#L72-L80

`mode` is a bitmap, not just a counter.  There might be times where setting `mode = 4` will trigger `(mode & MODE_UART) == 0` or `(mode & MODE_RESET) == 0` when we don't want it to.

I'm not sure I have a good solution to this.

Reply:
Actually, treating `mode` as a bitmap, you might as well still allow this, and use it when detecting CTS.